src/cap-description.adoc

```diff
@@ -280,15 +280,13 @@ it via a superset capability with <<CBLD>>. {cheri_base_ext_name} does not offer
 an unseal instruction.
 
 For code capabilities, the sealing bit is used to implement immutable
-capabilities that describe function entry points. Such capabilities can be leveraged
-to establish a form of control-flow integrity between mutually distrusting code. These
-capabilities are known as sealed entry (sentry) capabilities. A program may jump to a
-sentry capability to begin executing the instructions it references. The jump
-instruction automatically unseals the capability and installs it to the
+capabilities that describe function entry points, known as sealed entry (sentry) capabilities. Such capabilities can be leveraged
+to establish a form of control-flow integrity between mutually distrusting code. A program may jump to a
+sentry capability to begin executing the instructions it references. The <<JALR>>
+instruction automatically unseals a sentry target capability with zero offset and installs it to the
```

Comment:
I would turn around this sentence to make it a little clearer:
> A <<JALR>> instruction with zero offset automatically unseal a sentry target capability and installs it to the program counter...

Reply:
done